.husky/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+OUTPUT=$(npx lint-staged > /dev/null && echo $? || echo $?)
+
+if [ $OUTPUT -eq 0 ]; then
+    echo "\n\n✅ Lint Check Passed. ✅\n\n"
+    exit 0
+else
+    echo "\n\n❌ Lint Check failed... Use the uds task: lint-fix to resolve issues and re-commit. ❌\n\n"
```

Comment:
```suggestion
    echo "\n\n❌ Lint Check failed... Run \`uds run lint-fix\`  to resolve issues and re-commit. ❌\n\n"
```
This might be a more "beginner friendly" output (just give the exact command)?